src/Conf.ml

```diff
@@ -1875,18 +1875,30 @@ let is_in_listing_file ~quiet ~listings ~filename =
               |> List.filter ~f:(fun (_, l) -> not (drop_line l))
             in
             List.find_map lines ~f:(fun (lno, line) ->
-                if String.equal line "*" then Some (listing_file, lno)
-                else
-                  match Fpath.of_string line with
-                  | Ok file_on_current_line ->
-                      let f = Fpath.(dir // file_on_current_line) in
-                      if Fpath.equal filename f then Some (listing_file, lno)
-                      else None
-                  | Error (`Msg msg) ->
-                      if not quiet then
-                        Format.eprintf "File %a, line %d:\nWarning: %s\n"
-                          Fpath.pp listing_file lno msg ;
-                      None))
+                match Fpath.of_string line with
+                | Ok file_on_current_line -> (
+                    let f = Fpath.(dir // file_on_current_line) in
+                    if Fpath.equal filename f then Some (listing_file, lno)
+                    else
+                      try
+                        let re = Re.Glob.glob ~pathname:true line in
```

Comment:
You should set `~⁠anchored:true` otherwise `sr` will match `src/Abc.ml`.

Reply:
Done, also I used the wrong variable for the filename (fixed as well)